pyproject.toml

```diff
@@ -8,17 +8,18 @@ build-backend = "hatchling.build"
 [project]
 name="neurodamus"
 authors = [
-    {name = "Blue Brain Project, EPFL"},
```

Comment:
I wonder whether we should not keep BBP as author and add OBI as both author and maintainer?

Reply:
@mgeplf  @jdcourcol ?

Reply:
yes